bigframes/functions/remote_function.py

```diff
@@ -800,9 +898,25 @@ def remote_function(
             function's default setting applies. For more details see
             https://cloud.google.com/functions/docs/configuring/max-instances
     """
-    if isinstance(input_types, type):
+    is_row_processor = False
+
+    import bigframes.series
+
+    if input_types == bigframes.series.Series:
```

Comment:
Should we also handle pandas.Series?

Reply:
Um, what would it mean for the user? We are position `remote_function` as a way to apply their custom code on BigFrames DataFrames. It's not meant for applying to pandas DataFram, so why bring that up?

Reply:
I notice pandas series was supported as a type in a later change https://github.com/googleapis/python-bigquery-dataframes/pull/717/files#diff-e4315dd3ecff62becbb3e104eaf9232f49553c94e3f8fec0c5108ab98a4256aa, so resolving this.